src/example_node/example_node/example.py

```diff
@@ -0,0 +1,86 @@
+import sys
+
+
+import rclpy # Import the package
+from rclpy.executors import ExternalShutdownException 
+from rclpy.node import Node
+
+from std_msgs.msg import String
+
+from lib.color_codes import ColorCodes, colorStr # Import yummy colors
+from lib.interface.robot_info import RobotInfo # Read data
+from lib.configs import MotorConfigs
+from lib.interface.robot_interface import RobotInterface # Send data
+
+
+class ExampleNode(Node):
+    
+    def __init__(self) -> None:
+        # Call the parent's constructor and pass in the node's name
+        super().__init__("my_example_node") 
+        
+        # Print to the console with the color blue. Regular print() does not work
+        self.get_logger().info(colorStr("Launching example_node node", ColorCodes.BLUE_OK))
+        
+        self.robotInfo = RobotInfo(self)
+        self.robotInterface = RobotInterface(self)
+        
+        self.drivebaseSpeed = 6.28 # 1.0 rev/sec
```

Comment:
add unit; rename to `self.drivebaseSpeedRadPerSec` or smth. or change the comment to `# radians per second`